executor/README.md

```diff
@@ -0,0 +1,115 @@
+Cook Executor
+=============
+
+The Cook executor is a custom executor written in Python.
+It replaces the default command executor in order to enable a number of features for both operators and end users.
+For more information on Mesos executors, see the "Working with Executors" section at http://mesos.apache.org/documentation/latest/app-framework-development-guide/
+
+For Users
+---------
+
+The cook executor supports a number of (completely optional) features beyond the standard Mesos command executor.
+The executor sends the following information to the scheduler:
+1. progress update messages from the task (at intervals of `progress-sample-interval-ms` ms, see below),
+2. the exit code of the task, and
+3. the sandbox location of the task.
```

Comment:
Actually, none of these are now optional. The executor will perform all of the following actions.